examples/dgmulti_3d/elixir_navierstokes_convergence.jl

```diff
@@ -0,0 +1,253 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal compressible Navier-Stokes equations
+
+prandtl_number() = 0.72
+mu() = 0.01
+
+equations = CompressibleEulerEquations3D(1.4)
+equations_parabolic = CompressibleNavierStokesDiffusion3D(equations, mu=mu(), Prandtl=prandtl_number(),
+                                                          gradient_variables=GradientVariablesPrimitive())
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+dg = DGMulti(polydeg = 3, element_type = Hex(), approximation_type = Polynomial(),
+             surface_integral = SurfaceIntegralWeakForm(flux_lax_friedrichs),
+             volume_integral = VolumeIntegralWeakForm())
+
+top_bottom(x, tol=50*eps()) = abs(abs(x[2]) - 1) < tol
+is_on_boundary = Dict(:top_bottom => top_bottom)
+mesh = DGMultiMesh(dg, cells_per_dimension=(8, 8, 8); periodicity=(true, false, true), is_on_boundary)
+
+# Note: the initial condition cannot be specialized to `CompressibleNavierStokesDiffusion3D`
+#       since it is called by both the parabolic solver (which passes in `CompressibleNavierStokesDiffusion3D`)
+#       and by the initial condition (which passes in `CompressibleEulerEquations3D`).
+# This convergence test setup was originally derived by Andrew Winters (@andrewwinters5000)
+function initial_condition_navier_stokes_convergence_test(x, t, equations)
```

Comment:
And call it `initial_condition_convergence_test` in that case?

Reply:
The problem with renaming it is that it is not typed on the `equations`, thus it would trigger for any system of equations that is used and that does not have it implemented.

Reply:
Ah, I see, thanks!

Reply:
Should I move the `initial_condition_navier_stokes_convergence_test` and its corresponding sources terms function into the file (in 2D and 3D)? It would make the elixirs cleaner and remove some duplicate code, but it does not fit the naming style because we cannot simply call them `convergence_test` due to the dispatching issue discussed above.

Reply:
No, it's fine to keep everything as it is.